examples/aws-kms-key-complete/README.md

```diff
@@ -0,0 +1,83 @@
+# encryption-at-rest-aws-kms-key - complete example
+
+_Note: you can see the full source code in the [github repository](https://github.com/terraform-mongodbatlas-modules/terraform-mongodbatlas-encryption-at-rest/tree/main/examples/aws-kms-key-complete)_
+
+This example sets up encryption at rest using an AWS KMS for your Atlas Project. It creates the encryption key in AWS KMS, an IAM role and policy so that Atlas can access the key, and enables encryption at rest for the Atlas Project. Finally, it creates a Cluster with encryption at rest enabled.
+
+## Usage
+
+- Set the following variable: 
+
+    - `project_id`: ID of the Atlas project
+
+- Set the following environment variables:
+
+    -  `export MONGODB_ATLAS_PUBLIC_KEY="<YOUR_PUBLIC_KEY>"`
+    -  `export MONGODB_ATLAS_PRIVATE_KEY="<YOUR_PRIVATE_KEY>"`
+    -  `export AWS_ACCESS_KEY_ID="<YOUR_ACCESS_KEY>"`
+    -  `export AWS_SECRET_ACCESS_KEY="<YOUR_SECRET_KEY>"`
+    -  `export AWS_REGION="<YOUR_REGION>"`
+
+- Run the following command to initialize your project:
+
+```bash
+$ terraform init
+```
+
+- Run the following command to review the execution plan:
+
+```bash
+$ terraform plan
+```
+
+- Run the following command to deploy your infrastructure:
+
+```bash
+$ terraform apply
+```
+
+## Considerations
+
+- In order to define the AWS KMS Key policy with the correct permissions, it should look like this:
+
```

Comment:
I think it is a bit strong wording.
Your AWS KMS Key policy must allow the IAM Role access, this can be done by: